cypress/integration/phase.spec.js

```diff
@@ -84,15 +83,26 @@ describe('Phase functionality', () => {
     };
     completePhaseForm(formValues);
 
-    // expect: required error on every field
     cy.contains('p.Mui-error', 'Invalid entry. End date must be at least 1 day after Start date');
   });
 
+  it('New phase cannot overlap with existing phases', () => {
+    navigateToForm();
+    const formValues = {
+      phaseName: 'Test overlaps',
+      startDate: '1991/01/01',
+      endDate: '1992/01/01',
+    };
+    completePhaseForm(formValues);
+
+    cy.contains('p.Mui-error', 'Conflict with 1 or more phases');
+  });
+
   it('MoH can edit the start date and end date of a phase', () => {
     navigateToEditForm();
 
-    cy.get('[name=Startdate]').clear().type(`{ctrl+v}2024/01/05`);
-    cy.get('[name=Enddate]').clear().type(`{ctrl+v}2027/01/04`);
+    cy.get('[name=Startdate]').clear().type(`{ctrl+v}1990/06/06`);
```

Comment:
Just realized, you should probably add a comment on why the `paste` is needed

Reply:
Added comments in all tests using this logic, also added a comment to another section where we're mocking a downArrow click.